plugins/playroom/src/playroom.config.ts

```diff
@@ -27,6 +34,37 @@ require('@babel/register')({
   extensions: ['.ts', '.tsx', '.jsx', '.js', '.mjs']
 });
 
+// TODO: remove this or change based on advice after PR
+interface PlayroomConfig {
+  components: string;
+  outputPath: string;
+  title?: string;
+  themes?: string;
+  widths?: number[];
+  snippets?: Snippet[] | string;
+  frameComponent?: string;
+  exampleCode?: string;
+  cwd?: string;
+  storageKey?: string;
+  webpackConfig?: () => void;
+  baseUrl?: string;
+}
```

Comment:
I'm new to TypeScript and also want to follow conventions of the project so I'd like some guidance. Please let me know if I should define (or import?) the PlayroomConfig differently than this.

Reply:
If you can import it use that. But if they don't have a .d.ts file then you'll have to do this.

Looking at the code it seems like the already have this

https://www.runpkg.com/?playroom@0.18.1/src/index.d.ts

you could import it like

```ts
import. { PlayroomConfig }  from 'playroom/src';
```

Reply:
I was not able to import that config object so I left the interface definition in.